tests/keras_autolog/test_keras_autolog.py

```diff
@@ -149,8 +152,80 @@ def test_keras_autolog_logs_default_params(keras_random_data_run):
 @pytest.mark.parametrize('fit_variant', ['fit', 'fit_generator'])
 def test_keras_autolog_model_can_load_from_artifact(keras_random_data_run, random_train_data):
     run_id = keras_random_data_run.info.run_id
+    client = mlflow.tracking.MlflowClient()
     artifacts = client.list_artifacts(run_id)
     artifacts = map(lambda x: x.path, artifacts)
     assert 'model' in artifacts
     model = mlflow.keras.load_model("runs:/" + run_id + "/model")
     model.predict(random_train_data)
+
+
+@pytest.fixture
+def all_zeros_train_data():
+    return np.zeros((1000, 32))
+
+
+@pytest.fixture
+def all_zeros_one_hot_labels():
+    n, n_class = (1000, 10)
+    classes = np.zeros(n, dtype=int)
+    labels = np.zeros((n, n_class))
+    labels[np.arange(n), classes] = 0
+    return labels
+
+
+@pytest.fixture
+def keras_random_data_run_with_early_stop(all_zeros_train_data, fit_variant,
+                                          all_zeros_one_hot_labels, manual_run,
+                                          restore_best_weights):
+    mlflow.keras.autolog()
+
+    data = all_zeros_train_data
+    labels = all_zeros_one_hot_labels
+
+    model = create_model()
+    callback = keras.callbacks.callbacks.EarlyStopping(monitor='loss', patience=5,
```

Comment:
Thanks @srowen for the catch on the callback type check! @juntai-zheng let's add a preventative "regression" test against that bug by adding a test that runs with another unrelated callback, e.g. a `ProgbarLogger` callback (see how to construct one [from the Keras docs](https://keras.io/callbacks/)). We should be able to verify that the test fails without the fix & passes with the fix.